standards/3-commercial/com/usage.md

```diff
@@ -12,22 +12,30 @@ This property specifies the usage counters from which the service cost is calcul
 **Note:** This standard specifies a set of commonly applicable usage counters. It is expected from an execution environment on Provider node to support a certain set of usage counters (ie. be able to tally up the usage as the Activity is progressing). 
 
 ### **Examples**
-* `golem.com.usage.vector=["golem.usage.duration_sec"]` - Specifies 1-dimensional usage vector, consisting of Activity duration counter.
+* `golem.com.usage.vector=["golem.usage.duration-sec"]` - Specifies 1-dimensional usage vector, consisting of Activity duration counter.
 
 # Usage Counters
 The usage counters used to specify the service usage vector are defined in this section.
 
 **Note:** Usage counters are always expressed in Number type.
 
-## `golem.usage.duration_sec : Number (int32)`
+## `golem.usage.duration_sec : Number (int32)` [[Deprecated]](/standards/README.md#deprecated-properties)
```

Comment:
Great approach with marking those as deprecated 👍

Reply:
Good to deprecate these. But it breaks compatibility and we need to plan, how to change this at once in all places

Reply:
Deprecate means it still is supported. The old ones and the new ones in parallel. So backward compatibility should not break until old counters are removed.